src/main/java/dk/easv/helpers/AlertHelper.java

```diff
@@ -0,0 +1,35 @@
+package dk.easv.helpers;
+
+import dk.easv.be.Document;
+import dk.easv.dal.dao.DocumentDAO;
+import javafx.scene.control.Alert;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.concurrent.ConcurrentMap;
+import java.util.stream.Collectors;
+
+public class AlertHelper {
+    public static Alert alert;
+    public static void showDefaultAlert(String content, Alert.AlertType type) throws SQLException {
+        LocalDateTime currentDate = LocalDateTime.now();
```

Comment:
since this is only used once you can call the method straight

Reply:
called the alert inside controllers AdminView and ProjectManagerView